sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -542,16 +549,29 @@ public ServiceBusSenderClient buildClient() {
         private String sessionId;
         private String subscriptionName;
         private String topicName;
+        private Duration maxAutoLockRenewalDuration;
 
         private ServiceBusSessionReceiverClientBuilder() {
         }
 
+        /**
+         * Enables auto-lock renewal by renewing session lock renewal until the {@code maxAutoLockRenewalDuration} has
+         * elapsed.
```

Comment:
Explain little bit about `If there are multiple session` This would apply on each session separately.